### ISSUE TITLE
Twisted v17.1.0 fails on port string

### ISSUE THREAD

**Issue by robgjansen** [robgjansen/onionperf#26]
OnionPerf gives a warning when using twisted 17.1.0 that is not present on twisted 16.6.0:

```
[onionperf] [WARNING] command '/usr/local/bin/twistd -n -l - web --port
8090 --path /home/cloud/onionperf/onionperf-data/twistd/docroot
--mime-type=None' finished before expected
```

And the twisted log file `onionperf-data/twistd/onionperf.twisted.log`:

```
   app.run(runApp, ServerOptions)
 File
"/usr/local/lib/python2.7/dist-packages/twisted/application/app.py",
line 662, in run
   runApp(config)
 File
"/usr/local/lib/python2.7/dist-packages/twisted/scripts/twistd.py", line
25, in runApp
   _SomeApplicationRunner(config).run()
 File
"/usr/local/lib/python2.7/dist-packages/twisted/application/app.py",
line 380, in run
   self.application = self.createOrGetApplication()
 File
"/usr/local/lib/python2.7/dist-packages/twisted/application/app.py",
line 440, in createOrGetApplication
   ser = plg.makeService(self.config.subOptions)
 File "/usr/local/lib/python2.7/dist-packages/twisted/web/tap.py", line
254, in makeService
   strports.service(config['port'], site).setServiceParent(s)
 File
"/usr/local/lib/python2.7/dist-packages/twisted/application/strports.py",
line 40, in service
   endpoints.serverFromString(reactor, description), factory)
 File
"/usr/local/lib/python2.7/dist-packages/twisted/internet/endpoints.py",
line 1569, in serverFromString
   nameOrPlugin, args, kw = _parseServer(description, None)
 File
"/usr/local/lib/python2.7/dist-packages/twisted/internet/endpoints.py",
line 1487, in _parseServer
   getPlugins(IStreamServerEndpointStringParser), endpointType
 File
"/usr/local/lib/python2.7/dist-packages/twisted/internet/endpoints.py",
line 1503, in _matchPluginToPrefix
   raise ValueError("Unknown endpoint type: '%s'" % (endpointType,))
ValueError: Unknown endpoint type: '8090'
```

Reported by @hiromipaw (thanks!)

**Comment by hiromipaw** [comment]
Hi Rob,
As suggested, I run the twistd daemon separately and discovered that in version 17.1.0 the option for port was dropped (https://linux.die.net/man/1/twistd). Maybe this is the problem? 
Run:
```
$ /usr/local/bin/twistd -n -l - web  --path /onionperf/onionperf-data/twistd/docroot --port 8090 --mime-type=None
...
raise ValueError("Unknown endpoint type: '%s'" % (endpointType,))
ValueError: Unknown endpoint type: '8090'

```

**Comment by robgjansen** [comment]
Yes, you seem to be correct:
https://twistedmatrix.com/pipermail/twisted-python/2012-March/025362.html

Looks like we just need to find a work-around - I hope they have an alternative to that option.

**Comment by hadifarnoud** [comment]
did you find a workaround @robgjansen ? I'm using Sentry on Heroku. your solution doesn't work in Heroku

**Comment by robgjansen** [comment]
Well, I didn't recommend a solution, so I'm not surprised that it doesn't work (8-D) More seriously, though, we are planning on removing the twisted web server entirely from onionperf, so soon this issue will not affect us in this project.

**Comment by hiromipaw** [comment]
@hadifarnoud downgrading twisted version would work. It is a bit of a ugly hack though, I understand.

**Comment by jasco** [comment]
@robgjansen try `--port "tcp:port=8090"`

**Comment by irl** [comment]
This would be fixed by #45

**Comment by robgjansen** [comment]
Invalid, now that we removed support for the twistd server in ad3f234f7368f7e60ce7d798458c50e8ca0d8a35.
